### ISSUE TITLE
Can I write a MPC control algorithm to control Egocar by the Python API?

### ISSUE THREAD

**Issue by JackieLee-cn** [lgsvl/simulator#744]
![python-api-execution](https://user-images.githubusercontent.com/52964775/80897351-26f5b080-8d2a-11ea-8e22-8b252fb8b657.png)

**Comment by JackieLee-cn** [comment]
How can I control the Egocar. Do all control operations need to be implemented by the Apollo or autoware platform?

**Comment by EricBoiseLGSVL** [comment]
@JackieLee-cn Yes and no.  We have a simple tutorial for LaneFollowing but it is being updated for the new simulator currently.  See issue #698

**Comment by JackieLee-cn** [comment]
font{
        line-height: 1.6;
    }
    ul,ol{
        padding-left: 20px;
        list-style-position: inside;
    }


    
    Thank you very much for your help.Looking forward to the release of the next version as soon as possible.On 5/7/2020 05:27，Eric Boise<notifications
    
        
        

        
    

    On 5/7/2020 05:27，Eric Boise<notifications@github.com> wrote： 



@JackieLee-cn Yes and no.  We have a simple tutorial for LaneFollowing but it is being updated for the new simulator currently.  See issue #698
